apps/discovery_streams/lib/discovery-streams/stream_subscriber.ex

```diff
@@ -0,0 +1,28 @@
+# defmodule DiscoveryStreams.StreamSubscriber do
```

Comment:
Good riddance

Reply:
Lol this was a module I wrote.  Or maybe I replaced it with all of my code, but yea, it wasnt the right approach so this file will be deleted.

Reply:
Still. Good riddance ;)